pkg/graph/resolvers/it_gov_task_statuses.go

```diff
@@ -6,7 +6,24 @@ import "github.com/cmsgov/easi-app/pkg/models"
 
 // IntakeFormStatus calculates the ITGovTaskListStatus of a system intake for the requester view
 func IntakeFormStatus(intake *models.SystemIntake) models.ITGovIntakeFormStatus {
-	return models.ITGISReady
+	if intake.Step != models.SystemIntakeStepINITIALFORM {
+		return models.ITGISCompleted
```

Comment:
Did we decide not to do a `NOT NEEDED` status if a step gets skipped?

Reply:
I think we should still use the `NOT_NEEDED` status for skipped steps, but this specific step cannot be skipped according to FIGMA. ( Just any meeting or business case)

Reply:
Ah OK. I was wondering if that was the reason but just wanted to clarify, thanks.